tutorials/plugins/editor/making_plugins.rst

```diff
@@ -128,6 +126,15 @@ sometimes it can be cumbersome, especially if you're using it in many
 projects. A good solution to this is to make a plugin that adds a node with a
 custom behavior.
 
+.. warning::
+
+  Nodes added via an EditorPlugin are "CustomType" nodes. While they work
+  with any scripting language, they have fewer features than
+  :ref:`the Script Class system <doc_scripting_continued_class_name>`. If you
+  are writing GDScript or NativeScript, we recommend using them instead.
+  Godot 4.0 will have full Script Class support for all languages. At that
```

Comment:
I would not mention this, as it's not written in stone and we can't promise it will happen.